src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -415,6 +445,57 @@ private function getAbsoluteFilePath(string $filePath): string
         return sprintf('%s/%s', $this->getStubsDirectory(), $filePath);
     }
 
+    /**
+     * @param list<Node\Name> $nameNodes
+     *
+     * @return list<Node\Name>
+     */
+    private function modifyExtendsImplementsByPhpVersion(string $className, array $nameNodes): array
```

Comment:
This method does not modify any of the `$nameNodes`: its name is confusing, so we should probably call it something like `extendsOrImplementsReplacmentsByPhpVersion()` or such

Reply:
Changed. I hope it's ok.